classes/ActionScheduler_WPCLI_QueueRunner.php

```diff
@@ -39,13 +39,14 @@ public function __construct( ActionScheduler_Store $store = null, ActionSchedule
 	 * @author Jeremy Pry
 	 *
 	 * @param int    $batch_size The batch size to process.
+	 * @param array  $hooks      The hooks being used to filter the actions claimed in this batch.
 	 * @param string $group      The group of actions to claim with this batch.
 	 * @param bool   $force      Whether to force running even with too many concurrent processes.
 	 *
 	 * @return int The number of actions that will be run.
 	 * @throws \WP_CLI\ExitException When there are too many concurrent batches.
 	 */
-	public function setup( $batch_size, $group = '', $force = false ) {
+	public function setup( $batch_size, $hooks = array(), $group = '', $force = false ) {
```

Comment:
❔ I don't see this as an issue, but I'm just curious, why put hooks before `$group`?

Reply:
That was to try and match the order of params on `wc_schedule_single_action()` etc. (they're different param sets still, but `$hook` is before `$group` there, so figured it made sense to try to maintain that order here). :)